### ISSUE TITLE
Turkish Characters Support

### ISSUE THREAD

**Issue by gurayc** [rm-hull/luma.led_matrix#205]
Dear Developers,

Is it possible to add Turkish Chars like "ğüşıöçĞÜŞİÖÇ".

**Comment by rm-hull** [comment]
Yes it is possible to add bitmapped characters - see https://github.com/rm-hull/luma.core/pull/120 for example of how to do it - happy to accept a PR

**Comment by rm-hull** [comment]
Moving to luma.core
